LayoutTests/http/wpt/mediasession/voiceActivityDetection.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<body>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<iframe src="resources/toggleCapture-iframe.html" id=frame1></iframe>
+<iframe src="resources/toggleCapture-iframe.html" id=frame2></iframe>
+<video id=video autoplay playsinline></video>
+<script>
+
+promise_test(async () => {
+    return Promise.all([
+        new Promise(resolve => frame1.onload = resolve),
+        new Promise(resolve => frame2.onload = resolve),
+    ]);
+}, "Setup tests");
+
+promise_test(async (test) => {
+    if (!window.internals)
+        return;
+
+    video.srcObject = await frame1.contentWindow.startCapture(true, false);
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
With all of the `setTimeout`s in this test, it will take _at least_ 2.5 seconds to run. Can you wait for state changes instead of using hard coded timeouts?

Reply:
Done